package.json

```diff
@@ -16,7 +16,11 @@
     "lint": "ionic-app-scripts lint",
     "ionic:build": "ionic-app-scripts build",
     "ionic:serve": "ionic-app-scripts serve",
-    "meteor-client:bundle": "meteor-client bundle -s api -c meteor-client.config.json"
+    "meteor-client:bundle": "meteor-client bundle -s api -c meteor-client.config.json",
+    "tsc-compiler": "tsc",
+    "cordova-run": "ionic cordova run",
+    "android": "cd api/server && npm run tsc-compiler && cd ../.. && npm run cordova-run android -- ",
+    "ios": "cd api/server && npm run tsc-compiler && cd ../.. && npm run cordova-run ios -- "
```

Comment:
**Scripts added**
- "tsc-compiler" - to run the Typescript compiler in the given folder
- "cordova-run" - to tell ionic to run the app using cordova
- "android" - to run the application with the platform android
- "ios" - to run the application with the platform ios